.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+# Summary | Résumé
+
+> 1-3 sentence description of the changed you're proposing.
+> Please include the GitHub issue number or relevant acceptance criteria.
```

Comment:
Could add language on how to reference the issue number.  Possible add link to reference for more info.
Like `closes #000`
Reference: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword